extension/src/shared/view/fields/general/LayerDatasetStoryField.tsx

```diff
@@ -55,7 +55,6 @@ export const LayerDatasetStoryField: FC<LayerDatasetStoryFieldProps> = ({ atoms
             boundaryCount={1}
             onChange={handleChange}
           />
-          s
```

Comment:
😂

Reply:
Must be a cmd+s to save but cmd is not pressed....😂